pkg/actors/actors.go

```diff
@@ -901,6 +901,7 @@ func (m *ActorMetadata) calculateStateKey(actorType string, remindersPartitionID
 		"actors",
 		actorType,
 		m.ID,
+		"reminders",
```

Comment:
This is critical because it will keep this metadata storage extensible to support other information later on.